Packs/AtlassianConfluenceCloud/Integrations/AtlassianConfluenceCloud/AtlassianConfluenceCloud.py

```diff
@@ -1379,18 +1512,45 @@ def main() -> None:
             'confluence-cloud-space-list': confluence_cloud_space_list_command,
             'confluence-cloud-comment-create': confluence_cloud_comment_create_command,
             'confluence-cloud-content-create': confluence_cloud_content_create_command,
-            'confluence-cloud-space-create': confluence_cloud_space_create_command
+            'confluence-cloud-space-create': confluence_cloud_space_create_command,
         }
         command = demisto.command()
         args = demisto.args()
         strip_args(args)
         remove_nulls_from_dictionary(args)
+        limit = arg_to_number(params.get('max_events_per_fetch', 10000))
 
         if command == 'test-module':
             # This is the call made when pressing the integration Test button.
             return_results(test_module(client))
         elif command in commands:
             return_results(commands[command](client, args))
+
+        elif command == 'fetch-events':
+            last_run = demisto.getLastRun()
+            for events, next_link, last_end_date in fetch_events(client, last_run, limit):
+                demisto.info('events returned from fetch-events: ' + str(len(events)))
+                if events:
+                    add_time_to_events(events)
+                    send_events_to_xsiam(events, vendor=VENDOR, product=PRODUCT)
```

Comment:
dont update module health (should be an argument on this function)

Reply:
done